extensions/cursor/src/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Action, ActionPanel, Icon, List, openExtensionPreferences, getPreferenceValues } from "@raycast/api";
+import { useFetch } from "@raycast/utils";
+import { useState } from "react";
+import { URL } from "url";
+import type { DocumentationEntry } from "./types/types";
+
+export default function UserSearchRoot() {
+  const [search, setSearch] = useState<string>("");
+  const { isLoading, data, revalidate, error } = useFetch<DocumentationEntry[]>(
+    "https://cursor-raycast.degouville.com"
+  );
+
+  return (
+    <List
+      isLoading={isLoading}
+      searchBarPlaceholder="Search the Cursor Documentation"
+      onSearchTextChange={setSearch}
+      filtering={{ keepSectionOrder: true }}
+      throttle
+    >
+      <List.Section>
+        {(data || []).map((docsItem) => (
+          <List.Item
+            key={docsItem.title}
+            title={`${docsItem.title}`}
+            icon={docsItem.icon ?? "cursor-icon.png"}
+            actions={
+              <ActionPanel>
+                <Action.OpenInBrowser
+                  title="Open the Cursor Documentation"
+                  icon={"cursor-icon.png"}
+                  url={docsItem.url || ""}
+                />
+                <Action onAction={revalidate} title="Reload Items" icon={Icon.Download} />
+                <Action onAction={openExtensionPreferences} title="Open Extension Preferences" icon={Icon.Gear} />
```

Comment:
I'm not sure if it makes sense to show preferences if no preferences are available